src/Tribe/Ticket_Object.php

```diff
@@ -693,6 +738,18 @@ public function capacity() {
 				return '';
 			}
 
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 2c8f2e4e71fc231bd9c2c7c09ad3df10</sub>